python/setup.py

```diff
@@ -342,6 +342,7 @@ def get_packages(self):
         "aiosignal",
         "frozenlist",
         "requests",
+        "debugpy",
```

Comment:
We probably shouldn't install this by default; can we instead do a soft-import and raise a warning/exception if this is not installed?

Reply:
sure, will do